### PR TITLE
dictionary.json update

### LINKED ISSUE #287: add-meta scope incorrect
add-meta is allowed at widget and series level

image

### PR DESCRIPTION
**Pull request by annstriganova**
Resolves #287, #288, #289, #290

### CODE REVIEW

File: server/dictionary.json [hunk 1]
```diff
@@ -8253,7 +8253,7 @@
             "type": "string",
             "example": "Math.max(0, value('alias'))",
             "multiLine": true,
-            "section": "series",
+            "section": "option",
```
Comment (rodionos) [inline server/dictionary.json]: value can be specified in option AND series, as well as in other parent levels

Comment (annstriganova) [reply server/dictionary.json]: Yes, it works in this way:
selection_001

From `dictionary.schema.json`:

```
"section": {
            "description": "[Optional] Deepest section, where the setting can be applied. Default is global",
```

Comment (annstriganova) [reply server/dictionary.json]: From `validator.ts`:

```ts
    /**
     * Return true if the setting is allowed to be defined in the current section.
     * @param setting The setting to be checked.
     */
    private isAllowedInSection(setting: Setting): boolean {
        const currDepth: number = sectionDepthMap[this.currentSection.text];
        ...
        const reqDepth: number = sectionDepthMap[setting.section];
        return currDepth <= reqDepth;
    }
```

Comment (rodionos) [reply server/dictionary.json]: Will the validation work for both series and option?

Comment (annstriganova) [reply server/dictionary.json]: Yes, it will.

**Pull request merged**